spec/event_spec.rb

```diff
@@ -196,61 +197,69 @@
 
     describe 'FIRE' do
       it 'token empty' do
-        expect(faraday).to receive(:put).with("/v1/event/fire/#{event_name}", nil)
+        stub_request(:put, "http://localhost:8500/v1/event/fire/#{event_name}")
         Diplomat.configuration.acl_token = nil
-        ev = Diplomat::Event.new(faraday)
```

Comment:
great change, it makes test easier to read!

Reply:
Thank you!